README.md

```diff
@@ -1,78 +1,118 @@
+> `./node_modules/.bin/eslint --format codeframe pkg/commons-atom/ActiveEditorRegistry.js` (code from [nuclide](https://github.com/facebook/nuclide)).
 
-```
-$ npm install eslint-plugin-prettier --save-dev
-```
+## Installation
 
-**Note:** If you installed ESLint globally (using the `-g` flag) then you must also install `eslint-plugin-prettier` globally.
+```sh
+npm install --save-dev prettier eslint-plugin-prettier
+```
 
-## Usage
+**_`eslint-plugin-prettier` does not install Prettier or ESLint for you._** _You must install these yourself._
 
-Add `prettier` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
+Then, in your `.eslintrc`:
 
 ```json
 {
-    "plugins": [
-        "prettier"
-    ]
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": "error"
+  }
 }
 ```
 
+## Options
 
-Then configure the `prettier` rule under the `rules` section:
+* The first option:
 
-```json
-{
-    "rules": {
-        "prettier/prettier": "error"
-    }
-}
-```
+  - Objects are passed directly to Prettier as [options](https://github.com/prettier/prettier#api). Example:
+    
+    ```json
+    "prettier/prettier": ["error", {"singleQuote": true, "parser": "flow"}]
+    ```
 
-You can also pass [`prettier` configuration](https://github.com/prettier/prettier#api) as an option:
+  - Or the string `"fb"` may be used to set "Facebook style" defaults:
 
-```json
-{
-    "rules": {
-        "prettier/prettier": ["error", {"trailingComma": "es5", "singleQuote": true}]
-    }
-}
-```
+    ```json
+    "prettier/prettier": ["error", "fb"]
+    ```
 
-The rule will report an error if your code does not match `prettier` style. The rule is autofixable -- if you run `eslint` with the `--fix` flag, your code will be formatted according to `prettier` style.
+    Equivalent to:
 
----
+    ```json
+    "prettier/prettier": ["error", {
+      "singleQuote": true,
+      "trailingComma": "all",
+      "bracketSpacing": false,
+      "jsxBracketSameLine": true,
+      "parser": "flow"
+    }]
+    ```
 
-This plugin works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect patterns in the AST. (If another active ESLint rule disagrees with `prettier` about how code should be formatted, it will be impossible to avoid lint errors.) You can use [eslint-config-prettier](https://github.com/lydell/eslint-config-prettier) to disable all formatting-related ESLint rules. If your desired formatting does not match the `prettier` output, you should use a different tool such as [prettier-eslint](https://github.com/kentcdodds/prettier-eslint) instead.
+* The second option:
 
-## Migrating to 2.0.0
+  - A string with a pragma that triggers this rule. By default, this rule applies to all files. However, if you set a pragma (this option), only files with that pragma in the heading docblock will be checked. All pragmas must start with `@`. Example:
```

Comment:
To me, it seems like it would be better to get this functionality by using normal `eslint` directive comments, e.g.

```js
/* eslint prettier: ["error", "fb"] */
```

Is there a reason to have a special pragma for the rule?